src/bundle/Resources/public/js/scripts/fieldType/ezimage.js

```diff
@@ -43,7 +43,7 @@
             sizeContainer.title = fileSize;
 
             preview.querySelector('.ez-field-edit-preview__action--preview').href = URL.createObjectURL(files[0]);
-            this.fieldContainer.querySelector(SELECTOR_INPUT_ALT).dispatchEvent(new CustomEvent('cancelErrors'));
+            this.fieldContainer.querySelector(SELECTOR_INPUT_ALT).dispatchEvent(new CustomEvent('ez-cancel-errors'));
```

Comment:
`const` maybe?

Reply:
Moved to const: https://github.com/ezsystems/ezplatform-admin-ui/pull/956/commits/a5f8a64f1a22653049210ba9c7b1d95d9bd22e9b